docs/contributing/index.rst

```diff
@@ -117,17 +117,13 @@ Step 2: Create a Local Neighborhood Docker image
 
 Complete the ``Dockerfile`` in the
 ``docker-wrappers/LocalNeighborhood``
-directory to create a Docker image. The PathLinker ``Dockerfile``
+directory to create a Docker image. The AllPairs ``Dockerfile``
```

Comment:
The AllPairs Dockerfile does not actually demonstrate `COPY` so we will need to provide a different example or update this. COPY is needed for LocalNeighborhood.